renku/command/migrate.py

```diff
@@ -131,9 +131,9 @@ def _dockerfile_migration_check():
         Dictionary of Dockerfile migration status.
     """
     from renku import __version__
-    from renku.core.migration.migrate import is_docker_update_possible
+    from renku.core.migration.migrate import update_dockerfile
 
-    automated_dockerfile_update, newer_renku_available, dockerfile_renku_version = is_docker_update_possible()
+    automated_dockerfile_update, newer_renku_available, dockerfile_renku_version = update_dockerfile()
```

Comment:
Just an optional naming suggestion. `update_dockerfile` has a confusing name when you consider its only parameter actually skips the update and only checks. Seeing the function name at face value implies that the dockerfile will be updated and it does not indicate that it is possible to avoid overwriting the dockerfile. 

I propose 2 alternatvies to address this:
- Rename the function to clarify the intent - especially that the default is only a check OR
- Make the `check_only` function parameter be `False` by default force it to be used only with a keyword, i.e. `def update_dockerfile(*, check_only=False): ...`

I prefer the 2nd one. With the 2nd one you would have to say `update_checks = update_dockerfile(check_only=True)`. So the intent and consequences of the function are clearer.

EDIT: I realize after going through more code that the `update_dockerfile` function used to have the `check_only` parameter be `False` by default. I think this is another reason to not change it because someone may not realize we have changed it and introduce bugs.